rpcs3/Emu/Memory/vm.cpp

```diff
@@ -1370,10 +1370,9 @@ namespace vm
 			return {addr, nullptr};
 		}
 
-		// Special case
 		if (m_common)
 		{
-			return {addr, nullptr};
+			return {this->addr, m_common};
```

Comment:
What is the point of copying handle (m_common) that doesn't go anywhere?

Reply:
Carried on without it and added proper functionality in SPU side.